Formula/m/mklittlefs.rb

```diff
@@ -0,0 +1,21 @@
+class Mklittlefs < Formula
+  desc "Creates LittleFS images for ESP8266, ESP32, Pico RP2040, and RP2350"
+  homepage "https://github.com/earlephilhower/mklittlefs"
+  url "https://github.com/earlephilhower/mklittlefs/releases/download/4.1.0/mklittlefs-source.zip"
+  sha256 "4fa5565e2e9185898182557e25860f0fe5536f8133ef0247574105a75b5ae4d6"
+  license "MIT"
+
+  def install
+    system "make", "BUILD_CONFIG_NAME=-#{version}"
+    bin.install "mklittlefs"
+  end
+
+  test do
+    mkdir (testpath/"data")
+    (testpath/"data/hello.txt").write("Hello, World!")
+    (testpath/"data/Poetry.txt").write("Had we but world enough, and time...")
+    system(bin/"mklittlefs", "-s", "1048576", "-c", testpath/"data", testpath/"out.img")
+    system(bin/"mklittlefs", "-u", testpath/"out", testpath/"out.img")
+    system("diff", "-r", testpath/"data", testpath/"out")
```

Comment:
Let's add version check and minimize the test.

```suggestion
    assert_match version.to_s, shell_output("#{bin}/mklittlefs --version")

    (testpath/"data/hello.txt").write("Hello, World!")
    system bin/"mklittlefs", "-s", "1048576", "-c", "data", "out.img"
    system bin/"mklittlefs", "-u", "out", "out.img"
    system "diff", "-r", "data", "out"
```

Reply:
Thanks, but there's no need for the version check, it was just my initial attempt at a basic proof of life test.  If the build and verify work, we're all good.  (The diff command you replaced lost the `testpaths`, BTW.)

Reply:
Oh, and the 2 files are because one is Capitalized, the other lower case.  I wanted to make sure that we had case sensitive uploads (i.e. under Windows this might be an issue if MinGW isn't running right).  So I'd like to keep both files for that.